### PR TITLE
MC-12601: Added api changes for pricing when there is missing currencies

### PR DESCRIPTION
**Pull request by fyrnaga**
#### Changes made
* Added missing currencies on changes
* Added flag of missing currencies on change at the pricing level

#### Related PRs
- []()

### CODE REVIEW

File: source/includes/administration/_pricings.md [hunk 1]
```diff
@@ -152,6 +155,7 @@ Attributes | &nbsp;
 `name`<br/>*Map* | A map from language to name value for that language
 `description`<br/>*Map* | A map from language to description value for that language
 `supportedCurrencies`<br/>*List* | A list of currencies supported by the pricing
+`missingCurrenciesPricing`<br/>*boolean* | Specifies if one of the change is missing the prices for one or more currencies.
```
Comment (evanluc) [inline source/includes/administration/_pricings.md]:
```suggestion
`missingCurrenciesPricing`<br/>*boolean* | Specifies if one of the changes is missing the prices for one or more currencies.
```

File: source/includes/administration/_pricings.md [hunk 2]
```diff
@@ -438,6 +443,7 @@ Attributes | &nbsp;
...
```
Comment (evanluc) [inline source/includes/administration/_pricings.md]:
```suggestion
`missingCurrenciesPricing`<br/>*boolean* | Specifies if one of the changes is missing the prices for one or more currencies.
```

File: source/includes/administration/_pricings.md [hunk 3]
```diff
@@ -565,6 +575,7 @@ Attributes | &nbsp;
 `pricedProductsToModify`<br/> *Array<PricedProductChange>* | A change to a priced product in the pricing. Only with MODIFY_PRODUCTS and ADD_CURRENCIES type.
 `pricedProductsToDeprecate`<br> *Array<UUID>* | Priced products to deprecate
 `currenciesToAdd`<br/> *Array<String>* | Currencies to add to the pricing. Only with ADD_CURRENCIES type.
+`missingCurrencies`<br/> *Array<String>* | Currencies that are missing some price for either the unitPrice or COGS.
```
Comment (evanluc) [inline source/includes/administration/_pricings.md]:
```suggestion
`missingCurrencies`<br/> *Array<String>* | Currencies that are missing some prices for either the unitPrice or COGS.
```

**Review by evanluc — APPROVED** [review]

File: source/includes/administration/_pricings.md (hunk 1 above)
Comment (patrickspensieri) [reply source/includes/administration/_pricings.md]: Should also correct other places with `missingCurrenciesPricing` doc

**Review by patrickspensieri — APPROVED** [review]

File: source/includes/administration/_pricings.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: dc3d8906541b261c5eb88057b59ca74f4d619fb1

File: source/includes/administration/_pricings.md (hunk 2 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: dc3d8906541b261c5eb88057b59ca74f4d619fb1

File: source/includes/administration/_pricings.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: dc3d8906541b261c5eb88057b59ca74f4d619fb1

**Pull request merged**